src/components/App.js

```diff
@@ -1,6 +1,15 @@
 import React, { Component } from 'react'
 import styled from 'styled-components'
-import { BrowserRouter as Router, Route, NavLink } from 'react-router-dom'
+//import { BrowserRouter as Router, Route, NavLink } from 'react-router-dom'
+import Header from '../components/Header'
+import Card from '../components/Card'
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`
 
 export default class App extends Component {
   state = {
```

Comment:
Why is the toggleShowDoneTodos function still in here? If you don't need it, please delete it.